packages/contracts/contracts/voting/majority/MajorityVotingBase.sol

```diff
@@ -385,7 +388,7 @@ abstract contract MajorityVotingBase is
     /// @param _end The end date of the proposal vote. If 0, `_start + minDuration` is used.
     /// @return startDate The validated start date of the proposal vote.
     /// @return endDate The validated end date of the proposal vote.
-    function _validateVoteDates(uint64 _start, uint64 _end)
+    function _validateProposalDates(uint64 _start, uint64 _end)
```

Comment:
(Low importance)

I like to think of functions more like a "question" for which you get a response or like a script that does a batch of things for you. Validate does nothing or fails, I would write it like:
- `hasValidDates()` which replies yes/no and then you revert outside
- `assertValidDates()` or `assertDateRange()` to perform a clear action

Not important, feel free to resolve

Reply:
Changed it to `assertValidProposalDates` in 1cd94845a0fa51f7a09f78a1c530b18a2bfffeaf

Reply:
I don't like `assert` in the name of the function. `assert` is solidity keyword which refers to the fact that once it happens, INVALID opcode is thrown... putting it in the function name might be confusing and dev might be expecting that we actually do `assert()` in the function.

Reply:
@novaknole Pointed out that `assert` is a special keyword in solidity and that this is confusing to him.

Reply:
Reverted in dba4529aebdef6d79c696ce185cb529c096675de